<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RamChecker</class>
 <widget class="QWidget" name="RamChecker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>81</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="RamCheckLabel">
     <property name="text">
      <string>Client IP :&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="RamCheckEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
